cmd/utils.go

```diff
@@ -2,14 +2,85 @@ package cmd
 
 import (
 	"errors"
+	"fmt"
 	"path/filepath"
 	"strings"
 
 	"github.com/aerospike/asconfig/asconf"
+	"github.com/aerospike/asconfig/asconf/metadata"
 
 	"github.com/spf13/cobra"
 )
 
+const (
+	metaKeyAerospikeVersion = "aerospike-server-version"
+	metaKeyAsconfigVersion  = "asconfig-version"
+	metaKeyAsadmVersion     = "asadm-version"
+)
+
+var metaDataKeys = map[string]struct{}{
+	metaKeyAerospikeVersion: struct{}{},
+	metaKeyAsconfigVersion:  struct{}{},
+	metaKeyAsadmVersion:     struct{}{},
```

Comment:
hard coding seems like the wrong way to go. We agreed on a format and you have regex. Why not just copy them over and not have to worry about it? If I add more metadata on the asadm side will asconf need an update to support it?

Reply:
Maybe we don't care about further metadata

Reply:
I wanted to avoid writing comments that happen to meet the correct format to the file. I see your point about future items though, the table would need an update. I'll remove the filtering.

Reply:
removed it